package.json

```diff
@@ -77,6 +77,7 @@
         "js-yaml": "^3.13.1",
         "jsonpath-plus": "^4.0.0",
         "keycloak-connect": "^7.0.1",
+        "micromatch": "^4.0.2",
```

Comment:
I'm using this for the glob matching support in the instant launch mapping. infoType matching is also supported, and regex matching should be easy to add.